### ISSUE TITLE
"helm ls" doesn't show releases in PENDING_UPGRADE state

### ISSUE THREAD

**Issue by seanknox** [kubernetes/helm#2940]
STR:
- Upgrade a release using `helm upgrade --install --wait` that is unable to pull images

EXPECTED:
- `helm ls` would show the release in a PENDING_UPGRADE state

ACTUAL: 
- PENDING_UPGRADE is not listed, requires `helm ls -a` to see

While debugging a broken release upgrade, I was confused that the release no longer appeared in the cluster via `helm ls`. The release was broken due to `ImagePullBackOff` errors, and because I was using the `--wait` helm option, the Helm release would not show up until the Kubernetes deployment had actually failed.

```
$ helm ls
NAME                            	REVISION	UPDATED                 	STATUS         	CHART                   	NAMESPACE
customer                    	25      	Fri Sep 15 08:37:01 2017	DEPLOYED       	nginx-ingress-0.7.0     	kube-system
service                     	26      	Fri Sep 15 08:36:39 2017	DEPLOYED       	nginx-ingress-0.7.0     	kube-system

$ helm ls -a
NAME                            	REVISION	UPDATED                 	STATUS         	CHART                   	NAMESPACE
customer                    	25      	Fri Sep 15 08:37:01 2017	DEPLOYED       	nginx-ingress-0.7.0     	kube-system
service                     	26      	Fri Sep 15 08:36:39 2017	DEPLOYED       	nginx-ingress-0.7.0     	kube-system
my-release                      22      	Fri Sep 15 08:37:24 2017	PENDING_UPGRADE	foo-0.1.0               	my-release-westus2
```

**Comment by NocturnalShadow** [comment]
Why was this closed??? In helm 2.17.0 the issue still present. Was it fixed in Helm 3+?

**Comment by NocturnalShadow** [comment]
Never mind. I see adding `--all` flag to `helm ls` shows the real status of the release.
